Project-Sloth/ps-realtor
# [Fix] Updating type of price to number instead of text for manage property modal.

# Overview
Updating type of the price in the input field to number.

# Details
Today there is an error in the console if a realtor creates a house, then tries updating the price of it. This will cause the home to become unsellable. This resolves that by making the type of price a number instead of text.

# UI Changes / Functionality
Only change here is the text box would add the up and down arrows that numbers have.
![image](https://github.com/Project-Sloth/ps-realtor/assets/9836942/867e10ad-2320-46c6-aa14-7c9c5b59e5bf)

# Testing Steps
Old Repro steps would be creating a house, going to the manage screen and updating the price then trying to sell it. Without this change the sale will fail and you will see an F8 error about expecting a number and getting a string.
